packages/desktop-client/src/components/reports/reports/Calendar.tsx

```diff
@@ -0,0 +1,958 @@
+  calendar,
+  totalIncome,
+  totalExpense,
+  onApplyFilter,
+  firstDayOfWeekIdx,
+}: CalendarHeaderProps) {
+  return (
+    <View
+      style={{
+        minWidth: '300px',
+        maxWidth: '300px',
+        padding: 10,
+        borderRadius: 4,
+        backgroundColor: theme.tableBackground,
+      }}
+    >
+      <View
+        style={{
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+          flexWrap: 'wrap',
+          marginBottom: 16,
+        }}
+      >
+        <Button
+          variant="bare"
+          style={{
+            color: theme.pageTextSubdued,
+            fontWeight: 'bold',
+            fontSize: '14px',
+            margin: 0,
+            padding: 0,
+            display: 'inline-block',
+            width: 'max-content',
+          }}
+          onPress={() => {
+            onApplyFilter({
+              conditions: [
+                {
+                  field: 'date',
+                  op: 'is',
+                  value: format(calendar.start, 'yyyy-MM'),
+                  options: {
+                    month: true,
+                  },
+                },
+              ],
+              conditionsOp: 'and',
+              id: [],
+            });
+          }}
+        >
+          {format(calendar.start, 'MMMM yyyy')}
+        </Button>
+        <View
+          style={{ display: 'grid', gridTemplateColumns: '16px 1fr', gap: 2 }}
+        >
+          {totalIncome ? (
+            <>
+              <SvgArrowThickUp
+                width={16}
+                height={16}
+                style={{ color: chartTheme.colors.blue, flexShrink: 0 }}
+              />
+              <View
+                style={{
+                  color: chartTheme.colors.blue,
+                  flexDirection: 'row',
+                  flexGrow: 1,
+                  justifyContent: 'start',
+                }}
+                aria-label="Income"
+              >
+                <PrivacyFilter>
+                  {amountToCurrency(calendar.totalIncome)}
+                </PrivacyFilter>
+              </View>
+            </>
+          ) : (
+            ''
+          )}
+          {totalExpense ? (
+            <>
+              <SvgArrowThickDown
+                width={16}
+                height={16}
+                style={{ color: chartTheme.colors.red, flexShrink: 0 }}
+              />
+              <View
+                style={{
+                  color: chartTheme.colors.red,
+                  flexDirection: 'row',
+                  flexGrow: 1,
+                  justifyContent: 'start',
+                }}
+                aria-label="Expenses"
+              >
+                <PrivacyFilter>
+                  {amountToCurrency(calendar.totalExpense)}
+                </PrivacyFilter>
+              </View>
+            </>
+          ) : (
+            ''
+          )}
+        </View>
+      </View>
+      <div style={{ flexGrow: 1, marginBottom: 20 }}>
```

Comment:
Does View not work here?

Reply:
It works with display 'block', changed to it